components/button/button.css

```diff
@@ -0,0 +1,59 @@
+.btn{
+    padding: 0.5rem 0.75rem;
+    font-size: 1rem;
+    border-radius: 0.25rem;
+    border:none;
+}
+.btn-link{
+    padding: 0.5rem 0.75rem;
+    font-size: 1rem;
+    text-decoration: none;
+    border-radius: 0.25rem;
+}
+.btn-primary{
+    color: var(--background-color);
+    background-color: var(--primary-color);
+    border:0.125rem solid var(--primary-color);
+}
+.btn-secondary{
+    color: var(--secondary-color);
+    background-color: var(--background-color);
+    border:0.125em solid var(--secondary-color);
```

Comment:
Same units need to be used(either rem or em)

Reply:
thanks for poinint out! it is fixed now